managed/services/telemetry/telemetry.go

```diff
@@ -128,7 +128,8 @@ func (s *Service) Run(ctx context.Context) {
 			return
 		}
 
-		report := s.prepareReport(ctx)
+		dataSources := s.locateDataSources(s.config.telemetry)
```

Comment:
could you please move code `dataSources := s.locateDataSources(s.config.telemetry)` outside of doSend function? I guess it's not needed to init `dataSources` every time when doSend function is called because config file cannot be changed in the runtime.

Reply:
Done.